### ISSUE TITLE
Ubuntu ARM support

### ISSUE THREAD

**Issue by manbehindthemadness** [lutzroeder/netron#1047]
I am attempting to deploy onto an Nvidia AGX Orin - Ubuntu 2004 arm64 and it appears that the only arm64 environment supported is Apple Silicon:
```
Successfully built netron-6.5.3-py3-none-any.whl
npx electron-builder --mac --universal --publish never -c.mac.identity=null
  • electron-builder  version=23.6.0 os=5.10.104-tegra
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=dist/builder-effective-config.yaml
  ⨯ Cannot find module 'dmg-license'
[...]
make: *** [Makefile:52: build_electron] Error 1
(venv) tech@Artax:/usr/src/netron$ npm i dmg-license
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for dmg-license@1.0.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: arm64
```

**Comment by lutzroeder** [comment]
@manbehindthemadness the log is from running a macOS build on Ubuntu which is expected to fail.

Running `npx electron-builder --linux appimage --arm64 --publish never` is the command to create a Linux desktop image for arm64. Run `npm run start` to launch without electron-builder. Please submit pull requests with fixes if you find issues.

The web version at https://netron.app, `pip install netron` and Windows version have arm64 support as well.

**Comment by FunnyWii** [comment]
> The log is from running a macOS build on Ubuntu which is expected to fail.
> 
> * [x]  To create a Linux desktop image for arm64 run: `npx electron-builder --linux appimage --arm64`
> * [x]  To launch without electron-builder: `npm start`.
> * [x]  Web version at https://netron.app and `pip install netron` has arm64 support as well.
> * [ ]  Ubuntu Snap arm64 support is blocked by [electron-userland/electron-builder#5626](https://github.com/electron-userland/electron-builder/issues/5626).
> 
> Please submit pull requests with fixes if you find issues.

Hello Author
Via pip install netron, I see localhost:8080
But when I entried above address and launched the browser, the page is white and no UI is shown, that is, the page wasn't opened... The  https://netron.app has same problem.
I use firefox. I don't know whether I need specific browser?

**Comment by lutzroeder** [comment]
Duplicate of #1499
